pkg/kubelet/kubelet.go

```diff
@@ -1489,6 +1498,15 @@ func (kl *Kubelet) killUnwantedPods(desiredPods map[types.UID]empty,
 	return running, utilErrors.NewAggregate(errs)
 }
 
+func findPod(pods []*api.Pod, name string) *api.Pod {
+	for ix := range pods {
```

Comment:
added a TODO, though I'm really not sure there is measurable win here, given expected # of pods / machine.